crates/python/src/qpu/api.rs

```diff
@@ -539,6 +547,32 @@ py_wrap_type! {
     PyApiExecutionOptionsBuilder(ApiExecutionOptionsBuilder) as "APIExecutionOptionsBuilder"
 }
 
+#[pyclass(name = "QpuApiDuration")]
+#[derive(Debug, Clone)]
+pub struct PyQpuApiDuration {
+    inner: QpuApiDuration,
+}
+
+#[pymethods]
+impl PyQpuApiDuration {
+    #[new]
+    fn new(seconds: i64, nanos: i32) -> Self {
```

Comment:
Is there no "standard" duration type that PyO3 can use?

Reply:
They don't support `std::time::Duration`. They support `chrono::TimeDelta` but I don't think that's worth adding a new dependency.

Reply:
Sad day.